packages/source-iotsitewise/src/initialize.ts

```diff
@@ -45,6 +61,16 @@ export type SiteWiseQuery = {
 export const initialize = (input: SiteWiseDataSourceInitInputs) => {
   const siteWiseClient = getSiteWiseClient(input);
   const iotEventsClient = getIotEventsClient(input);
+  const store = new CreateTimeSeriesDataStore({
+    initialState: {
+      modeledDataStreams: [],
+      dataStreams: [],
+      thresholds: [],
+      assetModels: {},
+      alarms: {},
+      errors: {},
+    },
+  });
```

Comment:
Nice to have the creation of the data store explicitly occur in the initialization. Makes a lot of sense!